site/content/docs/5.0/components/badge.md

```diff
@@ -31,13 +35,45 @@ Note that depending on how they are used, badges may be confusing for users of s
 
 Unless the context is clear (as with the "Notifications" example, where it is understood that the "4" is the number of notifications), consider including additional context with a visually hidden piece of additional text.
 
+### Positioned
+
+Use utilities to modify a `.badge` and position it in the corner of a link or button.
+
+<!-- Boosted mod: replace the button with a button-icon because notifiers are only authorized with icons-->
+
 {{< example >}}
-<button type="button" class="btn btn-primary">
-  Profile <span class="badge bg-secondary">9</span>
-  <span class="visually-hidden">unread messages</span>
+<button type="button" class="btn btn-icon btn-secondary position-relative">
+  <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
+    <use xlink:href="/docs/5.0/assets/img/boosted-sprite.svg#buy"/>
+  </svg>
+  <span class="visually-hidden">Shopping basket</span>
+  <span class="position-absolute top-0 start-100 translate-middle badge rounded-pill bg-info text-white">
+    99+
+    <span class="visually-hidden">unread messages</span>
+  </span>
+</button>
+{{< /example >}}
+
+<!-- End mod -->
+
+You can also replace the `.badge` class with a few more utilities without a count for a more generic indicator.
+
+<!-- Boosted mod: replace the button with a button-icon because notifiers are only authorized with icons-->
+
+{{< example >}}
+<button type="button" class="btn btn-icon btn-secondary position-relative">
+  <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
+    <use xlink:href="/docs/5.0/assets/img/boosted-sprite.svg#buy"/>
+  </svg>
+  <span class="visually-hidden">Shopping basket</span>
+  <span class="position-absolute top-0 start-100 translate-middle p-2 bg-info rounded-circle">
```

Comment:
When removing `.badge` class (as explained) and using `.bg-info`, we can see that the blue color is different from the previous example:

![Capture d’écran 2021-07-01 à 10 50 58](https://user-images.githubusercontent.com/17381666/124095728-3e2d4c80-da5a-11eb-922f-73253bf54186.png)

![Capture d’écran 2021-07-01 à 10 51 26](https://user-images.githubusercontent.com/17381666/124095805-5309e000-da5a-11eb-9cb4-e71d4a71af83.png)

Seems a bit weird

Reply:
Modification in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/714/commits/c61ca96d02f56b0f506c428693a119196be0acce

I'm keeping this series of comments for the reviewer so he/she can see my process of thoughts.

Reply:
Discussions with the designers:

> [...] today in our framework, we do not have that type of component, a button that supports a notifier. The notifier only applies to an icon currently. As an example, the email icon with a notification of the number of unread.

> [..] currently we only provide this badge notification with the number so I would recommend just using this for now in terms of aligning the current standard

So I dropped the second example from the documentation (without the number) and used a link instead of a button.